package.json

```diff
@@ -905,22 +905,25 @@
 		"webpack-cli": "^3.1.2"
 	},
 	"dependencies": {
+		"@azure/arm-cosmosdb": "^3.0.0",
+		"@azure/ms-rest-azure-js": "^1.3.5",
+		"@azure/ms-rest-browserauth": "^0.1.4",
+		"@azure/ms-rest-js": "^1.8.7",
+		"@azure/ms-rest-nodeauth": "^2.0.0",
 		"antlr4ts": "^0.4.1-alpha.0",
-		"azure-arm-cosmosdb": "^1.1.2",
-		"azure-arm-resource": "^3.0.0-preview",
 		"d3": "^3.0.0",
 		"documentdb": "^1.14.2",
 		"event-stream": "3.3.4",
 		"fs-extra": "^4.0.2",
 		"gremlin": "^2.6.0",
+		"html-to-text": "^4.0.0",
 		"mongodb": "^2.2.25",
 		"mongodb-extended-json": "^1.10.0",
-		"ms-rest": "^2.2.1",
 		"node-uuid": "1.4.8",
 		"socket.io": "^1.7.3",
 		"socket.io-client": "^1.7.3",
 		"underscore": "^1.8.3",
-		"vscode-azureextensionui": "^0.24.0",
+		"vscode-azureextensionui": "file:../vscode-azuretools/ui/vscode-azureextensionui-0.25.0.tgz",
```

Comment:
Will change this to 0.25.0 once this PR is merged: https://github.com/microsoft/vscode-azuretools/pull/511